tests/ui/manual_str_repeat.rs

```diff
@@ -1,4 +1,4 @@
-#![allow(non_local_definitions)]
+#![allow(non_local_definitions, redundant_repeat_take)]
```

Comment:
Should be `clippy::redundant_repeat_take`. The test file also has errors because of that